tessera-jaxrs/thirdparty-jaxrs/src/main/java/com/quorum/tessera/thirdparty/PartyInfoResource.java

```diff
@@ -0,0 +1,53 @@
+package com.quorum.tessera.thirdparty;
+
+import com.quorum.tessera.partyinfo.PartyInfoService;
+import com.quorum.tessera.partyinfo.model.PartyInfo;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+import javax.json.Json;
+import javax.json.JsonArrayBuilder;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import static java.util.Objects.requireNonNull;
+
+@Path("/partyinfo")
+public class PartyInfoResource {
+
+    private final PartyInfoService partyInfoService;
+
+    public PartyInfoResource(final PartyInfoService partyInfoService) {
+        this.partyInfoService = requireNonNull(partyInfoService, "partyInfoService must not be null");
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Fetch network/peer information", produces = "public list of peers/publickey mappings")
```

Comment:
`produces` should be a comma-separated list of content-types

http://docs.swagger.io/swagger-core/v1.5.X/apidocs/index.html?io/swagger/annotations/ApiOperation.html

Reply:
good spot - it seems that this is the common error across the whole jaxrs apis. Suggest having them clean up in a separate PR